lstchain/scripts/lstchain_data_create_time_calibration_file.py

```diff
@@ -32,13 +32,13 @@
                     default=None, required=True)
 
 parser.add_argument("--output-file", action='store', type=str,
-                    dest='output_file'
+                    dest='output_file',
                     help="Path where script creates the time calibration file",
                     default=None, required=True)
 
 # Optional argument
 parser.add_argument("--max-events",
-                    dest='max-events'
+                    dest='max-events',
```

Comment:
I was going to change this in a new PR but I just realized this can be changed already here. Later in the script this variable is referred to as `max_events`.

Reply:
already changed in #395